.github/workflows/conventional-label.yaml

```diff
@@ -0,0 +1,14 @@
+# Warning, do not check out untrusted code with
+# the pull_request_target event.
+on:
+  pull_request_target:
+    types: [ opened, edited ]
+
+name: conventional-release-labels
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bcoe/conventional-release-labels@v1
+        with:
+          type_labels: '{"feat": "feature", "fix": "fix", "breaking": "breaking", "docs": "documentation", "ci": "ci"}'
```

Comment:
shouldn't dependencies be mapped here as well?

Reply:
Not now! Unless we want to start using `dependencies` as a commit type
But currently, the label is being added by dependabot itself :)

Reply:
the "build" message is usually what should be added when we're doing deps manually (happens every now and then, ex. when adding new ones)... probably good to add to the list anyway